versions/1/data_stream/spec.yml

```diff
@@ -39,14 +39,10 @@ spec:
         name: ingest_pipeline
         additionalContents: false
         contents:
-        - description: Default ingest pipeline definition
-          type: file
-          pattern: '^default\.(yml|json)$'
-          required: true
         - description: Supporting ingest pipeline definitions in YAML
           type: file
           pattern: '.+\.yml'
-          contentMediaType: "application/x-yaml"
+          contentMediaType: "application/x-yaml; require-document-dashes=true"
```

Comment:
Let's add a TODO or similar comment here referencing https://github.com/elastic/elasticsearch/issues/63247 so we know to revisit this change if the ES issue is resolved on their end.

Reply:
Added!